app/Http/Middleware/AdminMiddleware.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace App\Http\Middleware;
+
+use App\pronto\users\UserRoleManager;
+use Closure;
+
+class AdminMiddleware
```

Comment:
This works for now however we need a `RoleMiddleware` which takes an array of roles and checks if user has it
currently we want to support `admin`, `editor`, `member` roles